topologies/atedut.binding

```diff
@@ -27,6 +27,11 @@ duts {
     target: "dut-proxy-hostname:6030"
   }
 
+  # Options specific to gNOI
+  gnoi {
+    max_recv_msg_size: 1048576
```

Comment:
The unit is in bytes, so this sets the size to 1MB, not 1GB.

1GB is excessive. If we are sending more than 4MB in one response, we probably should explore other alternatives such as streaming gRPC.

Reply:
Done! Updated to 40MB which should suffice.